src/fw/apps/system_apps/settings/settings_system.c

```diff
@@ -374,7 +374,9 @@ static void prv_information_window_push(SettingsSystemData *data) {
             "%s, v%u", i18n_get_locale(), i18n_get_version());
 
   info->subtitle_text[SystemInformationItemBtAddress]  = info->bt_mac_addr;
-  info->subtitle_text[SystemInformationItemFirmware]   = (char*)TINTIN_METADATA.version_tag;
+  info->subtitle_text[SystemInformationItemFirmware]   =
+    (char*) (strlen(TINTIN_METADATA.version_tag) >= 2
```

Comment:
If the C preprocessor allowed `#if sizeof(...) > 2`, this would have been a compile-time thing. I could still make it a compile-time path (if we add an extra define in git_version.auto.h.in). Worth the tiny amount of time (and flash space) saved?